x-pack/plugins/cases/public/components/cases_context/cases_context_reducer.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CreateCaseFlyoutProps } from '../create/flyout';
+
+export const initialCasesContextState = (): CasesContextState => {
```

Comment:
nit: As it is a function I think is better to rename it from `initialCasesContextState` to `getInitialCasesContextState`